identity/src/main/java/com/stripe/android/identity/networking/DefaultIdentityRepository.kt

```diff
@@ -57,6 +59,9 @@ internal class DefaultIdentityRepository @Inject constructor(
             url = "$BASE_URL/$IDENTITY_VERIFICATION_PAGES/${urlEncode(id)}",
             options = ApiRequest.Options(
                 apiKey = ephemeralKey
+            ),
+            params = mapOf(
+                APP_IDENTIFIER to context.packageName
```

Comment:
Can this create a memory leak? Maybe we should have `Application` instead of `Context` in the dependency graph.

Reply:
Created https://jira.corp.stripe.com/browse/IDPROD-7120 to track this in a follow up

Reply:
Seems important enough that we’d want to fix it before we merge. Can you make sure this uses `context.applicationContext.packageName` for the time being?

Reply:
done